docs/cli.md

```diff
@@ -80,186 +356,217 @@ $ oc get pods -s template=production
 
 This command will return only pods whose `labels` include `"template": "production"`
 
```

Comment:
Prompt users here to see `oc describe` and do the same for oc describe to `oc get`.